src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -201,6 +203,13 @@
       InputListLocation="$(TestDependencyListFilePath)"
       OutputListLocation="$(OutDir)assemblylist.txt"
      />
+      
+    <!-- Additional execution commands included for specific scenarios -->
+    <!-- Testing with AppX -->
+    <ItemGroup>
+      <AppXCopyCommands Condition="'$(TestAppX)' == 'true'" Include="xcopy &quot;%PACKAGE_DIR%$([System.String]::Copy('%(_TestCopyLocalIncludeByForce.SourcePath)').Replace('$(PackagesDir)',''))&quot; &quot;%EXECUTION_DIR%$([System.String]::Copy('%(_TestCopyLocalIncludeByForce.SourcePath)').Replace('$(PackagesDir)microsoft.xunit.runner.uwp\$(AppXRunnerVersion)\',''))*&quot; /y"></AppXCopyCommands>
```

Comment:
So the copy list is populated in IncludedFileForRunnerScript which includes _TestCopyLocalByFileNameWithoutDuplicates which does not allow duplicate files (our app shim and exe would be excluded). Also the copy list only copies flat files (those not in a subdirectory) into the execution directory. So XunitUwpRunner.exe and the AppXManifest.xml would be copied but Core\XunitUwpRunner.exe   and Core\AppXManifest.xml (the shim folder) would not. This was a workaround for supporting this scenario without modifying the runtests template behavior.